test/ttnn-jit/test_eltwise.py

```diff
@@ -240,8 +242,10 @@ def test_unary_op_dram(device, h, w, dtype, op):
         sin,
         ceil,
         floor,
+        bitwise_not,
+        logical_not,
         # Not supported in TTIRToD2M:
-        # gelu, logical_not, reciprocal cbrt, sign, erf, erfc, bitwise_not
```

Comment:
You're welcome to tackle logical_not here too, but I think it will require another op :)